src/components/oak.py

```diff
@@ -401,41 +398,11 @@ async def get_point_cloud(
         if not self.oak_cfg.sensors.stereo_pair:
             details = "Cannot process PCD. OAK camera not configured for stereo depth outputs. Please see module docs in app configuration card."
             raise MethodNotAllowed(method_name="get_point_cloud", details=details)
-
-        self.get_point_cloud_was_invoked = True
-
-        # By default, we do not get point clouds even when color and depth are both requested
-        # We have to reinitialize the worker/pipeline+device to start making point clouds
-        if not self.worker.user_wants_pc:
-            try:
-                self.logger.info("Configuring worker for getting point clouds")
-                self.worker.user_wants_pc = True
-
-                # Use the worker manager to properly restart the worker
-                # This avoids race conditions with the worker's pipeline
-                self.worker_manager.restart_atomic_bool.set(True)
```

Comment:
i hate this

Reply:
Ah just saw this change

Reply:
Is this thing actually doing anything in the pointcloud worker? or is it just polling whether the oak-d had a pointcloud invoked?

Reply:
Seems like the latter - this is a good design choice.